### PR TITLE
Fix wrong type annotation

### LINKED ISSUE #1: TypeScript error (TS1345)
There is an error when using `rateLimiter` the latest TypeScript with Deno (in which the library is checked).

The full error:

```text
error: TS1345 [ERROR]: An expression of type 'void' cannot be tested for truthiness.
    if (!keyCheck) {
         ~~~~~~~~
    at https://deno.land/x/grammy_ratelimiter@v1.1.4/rateLimiter.ts:39:10
```

Example reproduction code:

```ts
import "https://deno.land/x/grammy_ratelimiter@v1.1.4/rateLimiter.ts";
```

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Fixes #1 hopefully.

### CODE REVIEW

File: src/typesAndDefaults.ts [hunk 1]
```diff
@@ -52,13 +52,13 @@ export interface OptionsInterface<C extends Context, RT extends RedisType> {
...
+  keyGenerator?: (ctx: C) => string | undefined;
 }
 
 export const defaultOptions = {
   timeFrame: 1000,
   limit: 1,
   onLimitExceeded: (_ctx: Context, _next: NextFunction) => {},
   storageClient: "MEMORY_STORE",
-  keyGenerator: (ctx: Context) => ctx.from && ctx.from.id.toString(),
+  keyGenerator: (ctx: Context) => ctx.from === undefined ? undefined : && ctx.from.id.toString(),
```
Comment (roj1512) [inline src/typesAndDefaults.ts:63]:
```suggestion
  keyGenerator: (ctx: Context) => ctx.from === undefined ? undefined : ctx.from.id.toString(),
```

**Review by roj1512 — CHANGES_REQUESTED** [review]

File: src/typesAndDefaults.ts (hunk 1 above)
Comment (KnorpelSenf) [reply src/typesAndDefaults.ts]: Haha thanks, coded on mobile ._.

**Review by roj1512 — APPROVED** [review]
Thanks, it works!

**Comment by KnorpelSenf** [comment]
I'll merge and release a fix tomorrow (unless you're faster @Amir-Zouerami).

**Comment by Amir-Zouerami** [comment]
> I'll merge and release a fix tomorrow (unless you're faster @Amir-Zouerami).

Fastest hands in the west! Btw, thanks for the pull request. I appreciate it.

**Comment by roj1512** [comment]
Haha, there are fast hands in the East, too.

**Pull request merged**